kview/src/main/java/dev/ikm/komet/kview/mvvm/view/details/DetailsController.java

```diff
@@ -900,6 +900,7 @@ private void setupDisplayUUID(EntityFacade entityFacade, ViewCalculator viewCalc
         String idStr = String.join(", ", idList);
 
         identifierControl.setPublicId(idStr);
+        identifierControl.setStyle("-fx-font-weight: normal;");
```

Comment:
can we set this in the css file rather than like this? Supposedly the identifier control text (not the label that goes left) is already NOT bold.